### ISSUE TITLE
Can you support an auto install ini config

### ISSUE THREAD

**Issue by plainee** [YidaozhanYa/Future-cyan-cursors-Windows#1]
Can you support an auto install ini config

**Comment by fmoricz** [comment]
I agree, an install.inf file would be needed for the installation on Windows.
You probably have it, you just forgot to add to the commit.
